examples/minimal_hand/visualize/mano_handstate.py

```diff
@@ -0,0 +1,309 @@
+"""Mesh representation of the MANO hand model.
+See `here <https://mano.is.tue.mpg.de/>`_ for details on the model.
+Their code has been refactored and documented here from Dr. Alexander Fabisch / DFKI-RIC.
```

Comment:
`Alexander Fabisch (DFKI GmbH, Robotics Innovation Center)` without the Dr. please

Reply:
and "has been refactored and documented by ..."